src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: rgba(0, 172, 220, 1);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, system-ui;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' is not available. For example, 'Arial' or another sans-serif font can be a good alternative.